hack/lib/serverless.bash

```diff
@@ -196,7 +196,7 @@ EOF
   logger.success 'KnativeServing has been updated successfully.'
 
   # metadata-webhook adds istio annotations for e2e test by webhook.
-  oc apply -f https://raw.githubusercontent.com/nak3/metadata-webhook/main/examples/release.yaml
+  oc apply -f "${rootdir}/serving/metadata-webhook/config"
```

Comment:
nit (maybe for a subsequent PR): We should probably replace the image that's deployed here with the version of the build itself. Currently, we'll always deploy the version at HEAD, so changes won't be immediately reflected.